patterns/2-structured/misusing metrics.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+Misusing Metrics
+
+## Patlet
+
+Metrics can be misused causing friction across teams. In these situations, create a sense of ownership within teams on how to use the data, avoiding comparisons, and begin measuring progress with deltas.
+
+## Problem
+
+The existence of metrics may lead to undesired situations and frictions across teams. Gamifications, list of rock stars or any other way of having people or teams “better” than others usually brings into the discussion defensive behaviors where the metric needs to be deprecated.
+
+## Context
+
+There is often a competitive spirit within development teams. In these situations, metrics tend to be used as evidence that supports a “better than” culture. This culture is a barrier to the healthy use of metrics to inform the successful progress in development velocity.
+
+## Forces
+
+* Individuals or teams may not understand the purpose of the metrics being used
+* Individuals or teams may feel threatened by how the metrics are being used
+* Leadership doesn’t understand how to utilize the metrics in a healthy way
+
+## Solutions
+
+Get a team internal agreement about the metrics that make sense in that specific team. Build a sense of ownership so everyone has contributed to the definition of the metric. Educate people about the right use of metrics and avoid comparisons. Measure deltas instead of net numbers, and look for outliers and how far projects are from the average instead of stating some behavior is “good” or “bad”.
+
+## Resulting Context
+
+Everyone being tracked and ongoing tracking are aligned into the existing and needed metrics. These metrics are publicly documented and anyone can learn from them. An existing set of metrics are part of the usual work of the teams with the goal of improving efficiency.
```

Comment:
This sounds more like it belongs into the **Solutions** section.

Quoting from the pattern template, this section should focus on:
> What is the situation after the problem has been solved? The original context is changed indirectly by way of the solution. 

So e.g. if in the **Problem** is that the unhealthy competition between teams was toxic, what is the behavior of the teams after the **Solution** is in place?

Reply:
Good point. Then this "By building a common understand, everyone develops a sense of ownership because they contributed to the definition of the metric. Understand and utilization of a common metrics language starts with the individual then built out to teams and finally the entire organization. This strengthened the internal culture around metrics as a tool for best development practices. " would be best for #resulting context.

Reply:
Yeah, probably.